app/src/org/commcare/tts/TextToSpeechConverter.kt

```diff
@@ -163,31 +187,6 @@ object TextToSpeechConverter {
             TextToSpeech.LANG_COUNTRY_VAR_AVAILABLE -> {
                 // Set language
                 tts.language = locale
-
-                // Check if voice data is present or not.
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-                    if (tts.voice != null) {
-                        val features = tts.voice.features
-                        if (features == null
```

Comment:
`features.contains(TextToSpeech.Engine.KEY_FEATURE_NOT_INSTALLED)` returns true irregularly even when the data is installed. Because of this false positive we were showing a dialog to install voice data when it was already installed.